src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -247,49 +38,56 @@ def __init__(
     ):
         super().__init__(path=path, code=code, dag=dag, **kwargs)
 
-        self._flow_dir, self._dag_file_name = self._get_flow_definition(self.code)
         self._executable = None
         self._params_override = params_override
+        self._flow_dir, self._dag_file_name = self._get_flow_definition(self.code)
 
-    @classmethod
-    def _load(cls, path: Path, dag: dict, **kwargs):
-        return cls(path=path, code=path.parent, dag=dag, **kwargs)
-
-    @property
-    def flow_dag_path(self) -> Path:
-        return self._flow_dir / self._dag_file_name
-
+    # region properties
     @property
     def name(self) -> str:
         return self._flow_dir.name
 
     @property
-    def display_name(self) -> str:
-        return self._data.get("display_name", self.name)
+    def flow_dag_path(self) -> Path:
+        return self._flow_dir / self._dag_file_name
 
     @property
     def tools_meta_path(self) -> Path:
         target_path = self._flow_dir / PROMPT_FLOW_DIR_NAME / FLOW_TOOLS_JSON
         target_path.parent.mkdir(parents=True, exist_ok=True)
         return target_path
 
+    @property
+    def language(self) -> str:
+        return self._data.get(LANGUAGE_KEY, FlowLanguage.Python)
+
+    @property
+    def additional_includes(self) -> list:
```

Comment:
This `additional includes` gets defined  both in `Flow` and `EagerFlow`, can we move it to base class?

Reply:
as well as `languange`